### DISCUSSION TITLE
Scrollable Widget

### DISCUSSION THREAD

**Discussion by securitybreach** [Docile-Alligator/Infinity-For-Reddit#917] [Ideas]
Could you please add a scrollable widget? I love the application but it would be even more perfect if it had one. Thanks and keep up the great work!

**Comment by Docile-Alligator** [comment]
Hi that's a good idea! Thank you!

**Comment by Wilker-uwu** [comment]
(remember to rename the issue title to be more descriptive \^\^')

**Comment by KaKi87** [comment]
Hello, any news on this ? Thanks

**Comment by Maio-msdos** [comment]
Alternative idea along the same theme - Home screen widget which displays the top N posts (N depending on the widget size) of a selected sub reddit

Apollo IOS reddit client does has something similar

Would likely be easier to implement than an interactive scrollable widget :)

**Comment by dnet890** [comment]
I love this feature as well. Boost & BaconReader have it. Infinity should too.

**Comment by bartalta** [comment]
+1, I absolutely love the implementation of this in Apollo on iOS.
